integrationtests/base.go

```diff
@@ -0,0 +1,139 @@
+package integrationtests
```

Comment:
also now you can split the file into 

- `wginterfacemanager.go` to have WgUp/WgDown functions
- `remoterun.go` to have the RemoteRun function

Reply:
can i add `GenerateSSHKeyPair`  to `remoterun.go` to the `base.go` just the functions which check connection?